package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "lemmy-js-client",
-  "version": "0.19.2-alpha.3",
+  "version": "0.19.2-alpha.4",
```

Comment:
Should the version for this still be 0.19.2? I would expect 0.19.3-something.

Reply:
Fixed